test/1_stdlib/UnsafeRawPointer.swift

```diff
@@ -0,0 +1,231 @@
+// RUN: %target-run-simple-swiftgyb
+// REQUIRES: executable_test
+
+import StdlibUnittest
+
+var UnsafeMutableRawPointerExtraTestSuite =
+  TestSuite("UnsafeMutableRawPointerExtra")
+
+class Missile {
```

Comment:
I made a note to follow up on this because I should also change the original UnsafePointer tests. For now the tests are consistent.